nengo/__init__.py

```diff
@@ -40,5 +63,5 @@
 except AttributeError:
     pass
 
-__copyright__ = "2013-2017, Applied Brain Research"
+__copyright__ = "2013-2019, Applied Brain Research"
```

Comment:
`LICENSE.rst` should also be updated to `2019`. Using the need to make #1522 as precedent, I split these changes off into #1523 and added a fixup commit here to revert this change.